pkg/kubelet/rkt/rkt.go

```diff
@@ -754,9 +755,15 @@ func podFinishedMarkerPath(podDir string, rktUID string) string {
 	return filepath.Join(podDir, "finished-"+rktUID)
 }
 
-func podFinishedMarkCommand(touchPath, podDir, rktUID string) string {
-	// TODO, if the path has a `'` character in it, this breaks.
-	return touchPath + " " + podFinishedMarkerPath(podDir, rktUID)
+func podFinishedMarkCommand(touchPath, podDir, rktUID string) (string, error) {
```

Comment:
since this is non-trivial now, you should add a unit test

Reply:
Sure. O.K

Reply:
Would you come and check it, if you're free. ^_^